TsRandomizer/LevelObjects/Other/GyrePortalEvent.cs

```diff
@@ -9,7 +9,36 @@ class GyrePortalEvent : LevelObject
 	{
 		public GyrePortalEvent(Mobile typedObject) : base(typedObject)
 		{
-			Dynamic._isUsable = false;
+			Dynamic._isUsable = true;
+			// Closed loop
+			if (typedObject.Level.ID == 14 && typedObject.Level.RoomID == 23)
+				Dynamic._portalType = 3; // post-boss
+			// Lab file cabinet room
+			else if (typedObject.Level.ID == 11 && typedObject.Level.RoomID == 4)
+            {
+				Dynamic._portalType = 2; // post-dungeon
+				LevelReflected.SetLevelSaveInt("GyreDungeonSeed", 0); // Warp to Ravenlord
+			}
+			// Backer room
+			else if (typedObject.Level.ID == 2 && typedObject.Level.RoomID == 51)
+            {
+				Dynamic._portalType = 2; // post-dungeon
+				LevelReflected.SetLevelSaveInt("GyreDungeonSeed", 1); // Warp to Ifrit
+			}
+
+			// TODO (currently have auto-warps on room entry)
```

Comment:
These dead lines of code should be removed

Reply:
Removed